test/arrays/views/viewPreserveDist.bad

```diff
@@ -1,4 +1,20 @@
-0.0 0.1 0.1 0.2 0.2 0.3 0.3
-0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
-0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
-0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
+D is: {1..8}
+X is: 0.0 0.0 0.1 0.1 0.2 0.2 0.3 0.3
+Y is: 0.0 0.0 0.1 0.1 0.2 0.2 0.3 0.3
+
+D is: {2..8}
+X is: 0.0 0.1 0.1 0.2 0.2 0.3 0.3
+Y is: 0.0 0.1 0.1 0.2 0.2 0.3 0.3
+
+D is: {1..8}
+X is: 0.0 0.0 0.0 0.0 0.2 0.2 0.2 0.2
+Y is: 0.0 0.0 0.0 0.0 0.2 0.2 0.2 0.2
+
+D is: {1..8}
+X is: 0.2 0.2 0.2 0.2 0.3 0.3 0.3 0.3
+Y is: 0.2 0.2 0.2 0.2 0.3 0.3 0.3 0.3
+
+D is: {0..15 by 2}
+X is: 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
+Y is: 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
+
```

Comment:
Yes, you're interpreting that correctly.  I could split it out, though the .bad should preserve the rank-change case working until reindex works (which should be any day now...).